src/test/forms/output/BenefitsLines.test.js

```diff
@@ -2,28 +2,28 @@ import React from 'react';
 import { mount } from 'enzyme';
 import { cloneDeep, set } from 'lodash';
 
-import { BenefitsLineGraph } from '../../../forms/output/BenefitsLineGraph';
+import { BenefitsLines } from '../../../forms/output/BenefitsLines';
 import { CLIENT_DEFAULTS } from '../../../utils/CLIENT_DEFAULTS';
 
-jest.mock('react-chartjs-2', () => {
-  const LineMock = () => {
-    return null;
-  };
+// jest.mock('react-chartjs-2', () => {
```

Comment:
Can we remove the commented lines?

Reply:
I thought they'd be worth keeping in case they could help us figure out highcharts stuff, but they're probably not worth it.